.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior with a reproducible whenever possible.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**System information (please complete the following information):**
+ - OS: `lsb_release -a`
+ - Avocado Version: `avocado -v`
+
+**Additional context**
```

Comment:
Maybe "Additional information"? Just a suggestion, if you take it, the text needs adjustment as well.